### DISCUSSION TITLE
Monitor problem with 21:9 scale and subtitles

### DISCUSSION THREAD

**Discussion by ghost** [mpv-player/mpv#14630] [General]
Hello, I have a 21:9 monitor and unfortunately when running movies, TV series - starts mpv with a rectangle in the center and with frames at the bottom, top and left, right. I managed to solve this problem with the following command: --panscan=1.0. However, I prefer to watch in subtitles and they disappear when this command is applied.

Who knows, please tell me how to adjust subtitles to the scale of the monitor 21:9 and to run the content on full screen?

**Comment by NoahIles** [comment]
pressing W worked for me
